bucket/python.json

```
@@ -1,34 +1,40 @@
 {
     "homepage": "https://www.python.org/",
-    "license": "https://docs.python.org/3/license.html",
+    "description": "A programming language that lets you work quickly and integrate systems more effectively.",
+    "license": "Python-2.0",
     "version": "3.7.3",
+    "depends": "dark",
     "architecture": {
         "64bit": {
-            "url": "https://www.python.org/ftp/python/3.7.3/python-3.7.3-amd64.exe#/py3.exe",
+            "url": "https://www.python.org/ftp/python/3.7.3/python-3.7.3-amd64.exe",
             "hash": "md5:a2b79563476e9aa47f11899a53349383"
         },
         "32bit": {
-            "url": "https://www.python.org/ftp/python/3.7.3/python-3.7.3.exe#/py3.exe",
+            "url": "https://www.python.org/ftp/python/3.7.3/python-3.7.3.exe",
             "hash": "md5:ebf1644cdc1eeeebacc92afa949cfc01"
         }
     },
     "installer": {
-        "args": [
-            "/quiet",
-            "InstallAllUsers=$(@{$true=1;$false=0}[$global])",
-            "TargetDir=\"$dir\"",
-            "AssociateFiles=0",
-            "Shortcuts=0",
-            "InstallLauncherAllUsers=$(@{$true=1;$false=0}[$global])"
-        ],
-        "keep": true
+        "script": [
+            "&(file_path dark dark.exe) -nologo -x \"$dir\\_tmp\" \"$dir\\$fname\" | Out-Null",
+            "@('launcher.msi', 'path.msi', 'pip.msi') | ForEach-Object {",
+            "    Remove-Item \"$dir\\_tmp\\AttachedContainer\\$_\"",
+            "}",
+            "(Get-ChildItem \"$dir\\_tmp\\AttachedContainer\\*.msi\").FullName | ForEach-Object { extract_msi $_ $dir }",
```

I will not have time in near future to test when quotes are needed. I would suggest to follow best practises and quote upper context variables, which contains whitespaces.
```suggestion
            "(Get-ChildItem \"$dir\\_tmp\\AttachedContainer\\*.msi\").FullName | ForEach-Object { extract_msi $_ \"$dir\" }",
```

---

Ok, add quotes now, and wait for further test.